code/modules/antagonists/abductor/abductor.dm

```diff
@@ -80,6 +80,8 @@
 /datum/antagonist/abductor/on_removal()
 	owner.special_role = null
 	REMOVE_TRAIT(owner, TRAIT_ABDUCTOR_TRAINING, ABDUCTOR_ANTAGONIST)
+	if(!isnull(owner.current))
+		REMOVE_TRAIT(owner.current, TRAIT_UNCONVERTABLE, ABDUCTOR_ANTAGONIST)
```

Comment:
That way we can put `TRAIT_UNCONVERTABLE` on the abductor's mind, not their body